backend/src/main.rs

```diff
@@ -23,11 +21,12 @@ use tokio::{
     signal::unix::{signal, SignalKind},
     sync::{mpsc, oneshot},
 };
+use once_cell::sync::Lazy;
```

Comment:
nit: Here and other places we would prefer the crate to be alphabetically placed in the `use` list. And one empty line after before the next section.

Reply:
Okay I will refactor accordingly, btw can next release contain any `CONTRIBUTE.md` file for easy instruction about guidelines

Reply:
> nit: Here and other places we would prefer the crate to be alphabetically placed in the `use` list. And one empty line after before the next section.

Does this applies to the stuff that we are importing or the start of the crate itself ?